src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -70,6 +71,12 @@ public class ShardRangeAssignment
             public int Shard;
         }
 
+        public Dictionary<int, BucketMigration> BucketMigrations;
+
+        public long MigrationCutOffIndex;
```

Comment:
what is the meaning of this?

Reply:
at some point we want to clean the old 'Pointers', so this is the cutoff index that will tell us upon which index to clean.
This will be also used to determine for the change vector when the `MOVE` tag is permanent.

Reply:
Okay, make sense. The idea is that we'll have a list of migrations sorted by dates, and we'll use that, right?

Reply:
To set the `MigrationCutOffIndex` 

Best to add a comment explaining this

Reply:
done